src/js/components/Form/__tests__/Form-test.js

```diff
@@ -96,7 +96,7 @@ describe('Form', () => {
   test('controlled lazy', () => {
     const onSubmit = jest.fn();
     const Test = () => {
-      const [value, setValue] = React.useState({});
+      const [value, setValue] = React.useState({ test: '' });
```

Comment:
Few words why this change fixes the warnings?

Reply:
If the value is undefined the input is uncontrolled. Then in line 100 when we try to assign a value to it the input becomes controlled which causes the warning: `A component is changing an uncontrolled input of type undefined to be controlled.` Setting test to an empty string initially makes the input controlled instead of uncontrolled.